swarm/storage/ldbstore.go

```diff
@@ -659,6 +662,7 @@ func (s *LDBStore) tryAccessIdx(ikey []byte, index *dpaDBIndex) bool {
 }
 
 func (s *LDBStore) Get(key Key) (chunk *Chunk, err error) {
+	//log.Trace("ldbstore.get", "key", chunk.Key) - seems like chunk is nil sometimes
```

Comment:
it is always nil, since it is a return value, named parameter initialised at zero value which is nil :)
you just meant to log `key`

Reply:
haha oh shit. yeah that makes sense :D